react/test/reactTextMask.spec.js

```diff
@@ -77,15 +77,6 @@ describe('MaskedInput', () => {
     expect(renderedDOMComponent.value).to.equal('')
   })
 
-  it('createTextMaskInputElement is a function', () => {
-    const maskedInput = ReactTestUtils.renderIntoDocument(
-      <MaskedInput
-      mask={['(', /\d/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]}
-      guide={true}/>
-    )
-    expect(typeof maskedInput.createTextMaskInputElement).to.equal('function')
-  })
-
```

Comment:
This isn't needed because createTextMaskInputElement is called directly, see the initTextMask function